File: 2026-09/src/day_12/prms_181862/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<string> solution(string myStr) {
    for (char& c : myStr) {
        if (c == 'a' || c == 'b' || c == 'c') c = ' ';
    }

    stringstream ss(myStr);
    vector<string> ans;
    string token;
    while (ss >> token) ans.push_back(token);

    if (ans.empty()) return {"EMPTY"};
    return ans;
}

File: 2026-09/src/day_12/prms_181862/Solution.java
package day_12.prms_181862;

import java.util.*;

class Solution {
    public String[] solution(String myStr) {
        StringTokenizer st = new StringTokenizer(myStr, "abc");
        if (!st.hasMoreTokens()) return new String[]{"EMPTY"};

        String[] arr = new String[st.countTokens()];
        for (int i = 0; i < arr.length; i++) {
            arr[i] = st.nextToken();
        }

        return arr;
    }
}
